src/helm/config/model_deployments.yaml

```diff
@@ -454,6 +454,24 @@ model_deployments:
       args:
         safety_settings_preset: block_none
 
+  - name: google/gemini-1.5-pro-002
+    model_name: google/gemini-1.5-pro-002
+    tokenizer_name: google/gemma-2b  # Gemini has no tokenizer endpoint, so we approximate by using Gemma's tokenizer.
+    max_sequence_length: 1000000  # Source: https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#gemini-models
+    # TODO: Max output tokens: 8192
+    client_spec:
+      class_name: "helm.clients.vertexai_client.VertexAIChatClient"
+
+  - name: google/gemini-1.5-flash-002
+    model_name: google/gemini-1.5-flash-002
+    tokenizer_name: google/gemma-2b  # Gemini has no tokenizer endpoint, so we approximate by using Gemma's tokenizer.
+    max_sequence_length: 1000000  # Source: https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#gemini-models
+    # TODO: Max output tokens: 8192
+    client_spec:
+      class_name: "helm.clients.vertexai_client.VertexAIChatClient"
+      args:
+        safety_settings_preset: default
```

Comment:
Delete this; if `safety_settings_preset` is not specified, then the safety thresholds are set to `BLOCK_NONE` as with the description. (Unfortunately, we don't default to `default` due to historical reasons...)

Reply:
Sounds good. 

I took a closer look at the [Gemini API doc](https://ai.google.dev/gemini-api/docs/safety-settings), `gemini-1.5-pro-002` and `gemini-1.5-flash-002` are actually set to different levels of safety settings depending on the specific API endpoint and [harm category](https://ai.google.dev/api/rest/v1/HarmCategory):

> If the threshold is not set, the default block threshold is Block most (for gemini-1.5-pro-002 and gemini-1.5-flash-002 only) or Block some (in all other models) for all categories except the Civic integrity category.  
> 
> The default block threshold for the Civic integrity category is Block most when sending prompts using Google AI Studio, and Block none when using the Gemini API directly.

Should we capture this in `model_deployment.yaml` too? Or should we leave it empty (by deleting L472-473)?